massa-models/src/config/constants.rs

```diff
@@ -342,6 +342,15 @@ pub const ROLL_COUNT_TO_SLASH_ON_DENUNCIATION: u64 = 1;
 /// Maximum size of executed denunciations
 pub const MAX_DENUNCIATION_CHANGES_LENGTH: u64 = 1_000;
 
+//
+// Constants for Api
+//
+
+/// Delta to compute upper bounds when fetching deferred credits
+pub const DEFERRED_CREDITS_DELTA: MassaTime = MassaTime::from_millis(
```

Comment:
this should be in the config file, as it is user-tunable

Reply:
Done